src/stores/admin.tsx

```diff
@@ -38,3 +43,20 @@ export const useSecuritySettingGeneralSWR = (): responseInterface<ISecurityParam
     { revalidateOnFocus: false },
   );
 };
+
+export const useUserGroupSWR = ({ pagination }): responseInterface<IUserGroup[], Error> => {
+  return useSWR(
+    ['/user-groups', pagination],
+    (endpoint) => apiv3Get(endpoint, { pagination })
```

Comment:
pagination が正しく使えてないですね
```suggestion
    (endpoint,pagination) => apiv3Get(endpoint, { pagination })
```